hieradata/vagrant/common.yaml

```diff
@@ -16,6 +16,9 @@ profile::letsencrypt::dns_azure:
   tenant_id: "<subscription id>" # subscription id
   zones:
     trusted.ci.jenkins.io: "/subscriptions/xxx/dnszone/yyyy"
+# Per-host Datadog configuration
+datadog_agent::host: "vagrant.local"
```

Comment:
Was that necessary?

Reply:
yes:

> This PR was tested on a local VM vagrant machine for the presence of both settings in /etc/datadog_agent/datadog.yaml configuration file

Reply:
Oups sorry, I looked only the modified files, I've missed the pr description

Reply:
no problem, I assumed you were on your phone ;)